nwb_conversion_tools/datainterfaces/giocomoconverter.py

```diff
@@ -0,0 +1,43 @@
+from ..nwbconverter import NWBConverter
+from .imagingextractorinterface import SbxImagingInterface
+from .roiextractordatainterface import Suite2pSegmentationInterface
+from .giocomovrdatainterface import GiocomoVRInterface
+from pathlib import Path
+import warnings
+
+
+class GiocomoImagingInterface(NWBConverter):
```

Comment:
Anything "Giocomo" should go in https://github.com/catalystneuro/giocomo-lab-to-nwb